readme.md

```diff
@@ -95,7 +95,7 @@ function setResponse(response) {
 ```
 
 ### To run
```

Comment:
Hi! Thank you for your contribution. Could you please also fix this line to be header at h2 level? Just doble hash instead of triple hash.